tools/pretty_printer.py

```diff
@@ -55,39 +50,28 @@ def trade_pretty_printer(trade):
             except KeyError:
                 order_type_name = trade.order_type.__class__.__name__
 
-        return "{0}: {1} {2} at {3} {4} on {5}: {6} ".format(
-            order_type_name,
-            PrettyPrinter.get_min_string_from_number(trade.quantity),
-            currency,
-            PrettyPrinter.get_min_string_from_number(trade.price),
-            market,
-            trade.exchange.get_name(),
-            convert_timestamp_to_datetime(trade.filled_time))
+        return f"{order_type_name}: {PrettyPrinter.get_min_string_from_number(trade.quantity)} {currency} " \
+            f"at {PrettyPrinter.get_min_string_from_number(trade.price)} {market} on {trade.exchange.get_name()}: " \
```

Comment:
I think that a [_Multi-line string syntax_](https://www.smallsurething.com/multi-line-strings-in-python/) will be better here. (with `"""` )